api/src/test/java/org/openmrs/validator/ConceptValidatorTest.java

```diff
@@ -528,7 +533,9 @@ public void validate_shouldNotFailIfBlankConceptDescriptionIsPassed() {
 		
 		concept.addName(new ConceptName("some name",Context.getLocale()));
 		concept.addDescription(new ConceptDescription("   ",null));
-		
+		concept.setConceptClass(new ConceptClass());
+		concept.setDatatype(new ConceptDatatype());
+
 		validator.validate(concept, errors);
 		
 		assertThat(errors, not(hasFieldErrors("description")));
```

Comment:
I need some help here, If the test says `shouldNotFail` shouldnt we checking for `not(HasErrors())` instead of checking for a specific field error?

Reply:
@LuGO0  the method name  (shouldNotFailIfAnyDescriptionIsNotEnteredWhileCreatingANewConcept) seems to talk about every thing,and checking for a specific field error is ok

Reply:
okay closing this then.

Reply:
it is well to add missing fields in concept validator